index.html

```diff
@@ -138,7 +138,20 @@ <h2>Find Me Resource</h2>
         Socials here
       </div>
       <div class="contents">
-        Contents here
+        <ul>
```

Comment:
More links to be added here. As the subjects start increasing. The link table shall be added